modules/ROOT/pages/addition/agent-installation/index.adoc

```diff
@@ -0,0 +1,329 @@
+= Installation and configuration of NOM agents
+:description: This section describes the installation process for a NOM agent.
+
+NOM agents should be installed for every host in the managed estate:
+
+* To manage a Neo4j instance, there needs to be a NOM agent running on the same host.
+* To fully manage a clustered DBMS - there must be an agent for every host that hosts an instance of the DBMS.
+* An agent can and should manage all instances on a single host even if they are members of different DBMSs.
+
+== NOM agent setup process
+* <<register,Register agent with NOM server>>
+* <<install,Install agent on Neo4j instance host>>
+* <<configure,Configure agent>>
+* <<running-agent,Run agent on host>>
+* <<verify,Verify agent setup>>
```

Comment:
I worry it might be a bit confusing that some of the info is on the root-level page (Installation and configuration of NOM agents) and some is on a sub-page (the registration stuff).

Maybe it would be clearer if the root-level page was just the content above this comment, essentially a a brief description and then a bullet list of steps, like:
1. Register agent
  There are two ways:
  a) Manual registration - recommended for manually..
  b) Self-registered - recommended for automated..
2. Configure and run agent

And then the bulk of the content on index.adoc is moved into the second bullet. Thoughts?

Reply:
OR maybe there should be 3 top-level sections like:

* Neo4j instance requirements
* Agent registration
* Agent configuration

Reply:
Yeah I am restructuring this section in a similar pattern, just weighing clarity against brevity, if duplicating same steps woun't hurt